app/controllers/auth/signUp.js

```diff
@@ -0,0 +1,38 @@
+const { User } = require('../../models');
+const { checkWoloxEmail, validateEmailExists, validatePassword } = require('../../../utils/validations');
+const hashPassword = require('../../../utils/hashPassword');
+const { NOT_BELONG_COMPANY, EMAIL_ALREADY_EXISTS, INVALID_PASSWORD } = require('../../errors');
+const logger = require('../../logger');
+
+exports.signUp = async (req, res) => {
+  const { name, lastName, email, password } = req.body;
```

Comment:
All input and output fields should be on `snake_case` and should be mapped to `camelCase`, I know this is kind of a pain but lets do it

Reply:
Fixed it